### PR TITLE
Custom check

### PR DESCRIPTION
**Pull request by ChristopherMancuso**
@RemyLau For #176 it was reading just the pre-stored networks, and I changed it to read anything in `file_loc` that has a node order

For #175 if a user had a custom network selected and auto_download on it raises an error, which I think is fine. I just tried to clear up that it is just a problem with trying to download the files.

### CODE REVIEW

File: geneplexus/download.py [hunk 1]
```diff
@@ -186,8 +186,10 @@ def _make_download_options_list(
 
     for i in opts:
         if i not in check_list:
-            raise ValueError(f"Unexpected {name}: {opts!r}")
-
+            raise ValueError(
+                f"Unexpected {name}: {opts!r} when downloading\n"
+                + "If using custom networks/GSCs turn auto_download off",
```
Comment (RemyLau) [inline geneplexus/download.py:191]: Could you elaborate about this note a little more (as well as #175)? I think if a user were to use custom networks, they are not going to try to "download" that network anyways, so why is this necessary?

Comment (RemyLau) [inline geneplexus/download.py:191]:
```suggestion
                f"Unexpected {name}: {opts!r} when downloading. "
                "If using custom networks/GSCs turn auto_download off",
```

**Comment by RemyLau** [comment]
@ChristopherMancuso I have proposed a new solution for fixing #175 (see https://github.com/krishnanlab/PyGenePlexus/pull/179/commits/737a4cf4f781977fcf10b975fedcf0ff0e793563). Instead of prompting error inside the private `_make_download_options_list` function, I think prompting warning messages during the initialization of the `geneplexus` object is better for the following reasons
1. It does not through error
1. It is more localized, i.e., it's easier for the user to find the `auto_download` option when the error comes from `geneplexus.__init__` rather than `_make_download_options_list`.

**Comment by ChristopherMancuso** [comment]
@RemyLau That is a much better way to fix #175. I think it all seems good now.
